### PR TITLE
refactor: Removed default database options and added as plain default.

### PR DESCRIPTION
**Pull request by rohitp-bsf**
### Description
refactor: Removed default database options and added as plain default.

### Checklist:
- [ ] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by vrundakansara — APPROVED** [review]

File: inc/core/builder/class-astra-builder-options.php [hunk 1]
```diff
@@ -301,13 +301,48 @@ function astra_hf_builder_customizer_defaults( $defaults ) {
...
+		'section-below-footer-builder-margin',
+		'section-footer-copyright-margin',
+		'section-footer-menu-margin',
+		'section-primary-footer-builder-padding',
+		'section-primary-footer-builder-margin',
+		'header-menu1-megamenu-heading-space',
+		'header-menu2-megamenu-heading-space',
+		'section-hb-language-switcher-margin',
+		'section-fb-language-switcher-margin',
+		'header-mobile-menu-submenu-spacing',
```
Comment (premanshup) [inline inc/core/builder/class-astra-builder-options.php:331]: @rohitp-bsf This setting is defined in the addon. Can you please check all the defaults which have settings added in addon should be moved to the addon?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/core/builder/class-astra-builder-options.php (hunk 1 above)
Comment (rohitp-bsf) [reply inc/core/builder/class-astra-builder-options.php:331]: Resolved: Added a PR for addon: https://git.brainstormforce.com/bsf-store-products/astra/astra-addon/-/merge_requests/2147

Comment (premanshup) [reply inc/core/builder/class-astra-builder-options.php:331]: @rohitp-bsf Thank you!

File: inc/core/builder/class-astra-builder-options.php [hunk 2]
```diff
@@ -301,13 +301,48 @@ function astra_hf_builder_customizer_defaults( $defaults ) {
...
+		'section-header-mobile-menu-margin',
+		'section-above-footer-builder-padding',
+		'section-above-footer-builder-margin',
+		'section-below-footer-builder-margin',
+		'section-footer-copyright-margin',
+		'section-footer-menu-margin',
+		'section-primary-footer-builder-padding',
+		'section-primary-footer-builder-margin',
+		'header-menu1-megamenu-heading-space',
+		'header-menu2-megamenu-heading-space',
```
Comment (premanshup) [inline inc/core/builder/class-astra-builder-options.php:328]: @rohitp-bsf Can you please remove them from here since they are already added in the addon?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/core/builder/class-astra-builder-options.php (hunk 2 above)
Comment (rohitp-bsf) [reply inc/core/builder/class-astra-builder-options.php]: Yes, @premanshup removed!

**Review by premanshup — APPROVED** [review]

**Pull request merged**